.github/workflows/load-tests.yaml

```diff
@@ -144,21 +164,13 @@ jobs:
       - name: Setup Load Testing environment
         run: |
           npm install
-          npm run setup:${K6_TOPOLOGY_NAME}
+          npm run setup
         working-directory: "./k6"
 
       - name: Run Load test
         run: |
           kubectl delete -f k6-test-run.yaml 2> /dev/null || true
```

Comment:
Would it make sense to create a crossplane resource called "load test" and deploy that, where individual steps of the deployment and post-deployment procedure would result in trackable side effects, that this job could monitor?